test/asyncio_tests/test_asyncio_collection.py

```diff
@@ -260,6 +266,79 @@ def test_sub_collection(self):
             self.assertEqual(read_preference, c.read_preference)
             self.assertEqual(codec_options, c.codec_options)
 
+    @env.require_version_min(6, 2, -1, -1)
+    @asyncio_test
+    async def test_async_create_encrypted_collection(self):
+        if pymongo.version_tuple < (4, 4, 0):
+            raise unittest.SkipTest("Requires PyMongo 4.4+")
+        c = self.collection
+        KMS_PROVIDERS = {"local": {"key": b"\x00" * 96}}
+        self.cx.drop_database("db")
+        async with AsyncIOMotorClientEncryption(
+            KMS_PROVIDERS, "keyvault.datakeys", c, CodecOptions()
+        ) as client_encryption:
+            coll, ef = await client_encryption.create_encrypted_collection(
+                database=self.db,
+                name="testing1",
+                encrypted_fields={"fields": [{"path": "ssn", "bsonType": "string", "keyId": None}]},
+                kms_provider="local",
+            )
+            with self.assertRaises(pymongo.errors.WriteError) as exc:
+                coll.insert_one({"ssn": "123-45-6789"})
```

Comment:
How is this working without the await? What is the type of `coll` here?

Reply:
This should be fixed now.

Reply:
How was it passing before?

Reply:
Before there were no awaits, so the function calls were returning immediately and not raising any exceptions.

Reply:
Apparently this was failing before.